core/src/main/java/bio/terra/pearl/core/model/file/ParticipantFile.java

```diff
@@ -28,4 +31,7 @@ public class ParticipantFile extends BaseEntity {
     private UUID enrolleeId;
 
     private String notes;
+
```

Comment:
I understand the need to send this association to the frontend, but I worry this obfuscates the underlying data model of response -> answer -> file too much. 
How about we make this a `List<Answer>` instead?  then those answers will still have the SurveyResponse id you need, and you'll also get other juicy tidbits too and a reminder aof how it's actually attached

Reply:
Also, while you're at it, please add a comment on the `List<ParticipantFile> `on SurveyResponse talking about how that relation is actually stored on the answers

Reply:
I think that will make the fetch logic simpler too -- instead of a nested for match and a new dao method in SurveyResponse, you'll just need to create a `findWithAnswers` method in ParticipantFileDao